blog/_posts/2017-08-03-Codeaholics,-the-first-hangover.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Codeaholics, the first Hangover!
+layout: post
+created_at: Thursday August 03 2017
+permalink: blog/2017-08-03-Codeaholics,-the-first-hangover
```

Comment:
while we don't have specific rules about commas in the filename + permalink, would it be possible for you to remove the comma here (and to rename the file accordingly)? That way we keep it simple :)